app/code/Magento/Ui/view/base/ui_component/etc/definition/ui_settings.xsd

```diff
@@ -960,13 +960,6 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
-            <xs:element name="required" type="translatableString">
```

Comment:
Backwards compatibility concern: Does removing this break any code that does have such a `<required>` tag defined still?

Reply:
Great catch. Thanks Ryan.
Removing the `<required>` tag will affect any existing code that relies on it. However, it’s worth noting that the implementation for this tag has not been implemented since its introduction in Magento version 2.2.0.

I’m considering whether to keep this XML validation or remove it and wait for this PR to be merged in a minor release (2.5.0 😅), where breaking changes are allowed. I’d love to hear your thoughts on this!